Firmware/mmu2_protocol_logic.cpp

```diff
@@ -510,12 +524,15 @@ ProtocolLogic::ProtocolLogic(MMU2Serial *uart, uint8_t extraLoadDistance)
     , progressCode(ProgressCode::OK)
     , buttonCode(NoButton)
     , lastFSensor((uint8_t)WhereIsFilament())
-    , regs8 { 0, 0, 0 }
-    , regs16 { 0, 0 }
-    , initRegs8 { extraLoadDistance }
     , regIndex(0)
-    , mmuFwVersion { 0, 0, 0 }
-{}
+    , retryAttempts(MAX_RETRIES)
+    , inAutoRetry(false) {
+    // @@TODO currently, I don't see a way of writing the initialization better :(
+    // I'd like to write something like: initRegs8 { extraLoadDistance, pulleySlowFeedrate }
```

Comment:
Could we pass a reference to a struct?

Reply:
yes, but it wouldn't help with the initialization of the `initRegs8` array